lldb/test/API/tools/lldb-server/TestGdbRemotePlatformFile.py

```diff
@@ -108,6 +109,20 @@ def test_platform_file_wronly_fail(self):
         )
         self.expect_gdbremote_sequence()
 
+    def remote_install(self, path, filename="test"):
+        if lldb.remote_platform:
+            remote_path = lldbutil.append_to_process_working_directory(self, filename)
+            err = lldb.remote_platform.Install(
+                lldb.SBFileSpec(path, True), lldb.SBFileSpec(remote_path, False)
+            )
+            if err.Fail():
+                raise Exception(
+                    "remote_platform.Install('%s', '%s') failed: %s"
+                    % (path, remote_path, err)
+                )
+            path = remote_path
+        return path
```

Comment:
This looks almost the same as the code in https://github.com/llvm/llvm-project/pull/91931, can this be de-duplicated?

Reply:
Sure. Please look at #91944.

Reply:
I have updated this patch using lldbutil.install_to_target().